### PR TITLE
adding functionality to append list of saved jobs, and ability to uns…

### PR DESCRIPTION
**Pull request by mwedgworth**
…ave jobs

### CODE REVIEW

**Review by typies — CHANGES_REQUESTED** [review]
New code in Pages.py is missing

File: User.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def __init__(self, first_name, last_name, username,
         self.experience = experience
         self.education = education
         self.previous_jobs = []
+        self.saved_jobs = []x
```
Comment (typies) [inline User.py:39]: Is this line supposed to end in an x?

File: State.py [hunk 2]
```diff
@@ -80,6 +80,20 @@ def save_jobs(self):
         else:
             print("All permitted jobs have been created, please come back later.\n")
             return False
+        
+
+    def saved_jobs(self):
```
Comment (typies) [inline State.py:85]: I think this is supposed to be "save_jobs" which is what is called in another file

File: Pages.py [hunk 3]
```diff
@@ -1062,12 +1070,31 @@ def apply(self, job_id):
...
+
+    def remove_saved(self, jobs):
+        # remove the current job from the student's list of saved jobs
+        self.state.jobs.remove(self.state.current_user.saved_jobs)
+        # save the changes to the system
         self.state.save_jobs()
+        print("\nThis job is no longer in your saved list.")
+        return   
+    def view_saved(self):
+        for job in self.state.current_user.saved_jobs:
```
Comment (typies) [inline Pages.py:1091]: Does this function do anything? It just looks like an empty for loop

**Review by typies — CHANGES_REQUESTED** [review]

File: State.py (hunk 2 above)
Comment (mwedgworth) [reply State.py:85]: no it is not, save_jobs does something else

**Review by SophiaPJones — APPROVED** [review]

**Review by typies — APPROVED** [review]

**Pull request merged**